docs/guide.rst

```diff
@@ -810,8 +810,8 @@ Current Limits
   `this issue`__ for more details.
 * interaction between guests provisioned by different plugins. Think
   "a server from ``podman`` plugin vs client from ``virtual``".
-  This is not yet supported, see these issues: `here`__
-  and `here`__.
+  This is not yet supported, see this issue: `here`__.
```

Comment:
Could the "this issue" be already the link please?

Reply:
done.